http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/BaseCloudServiceRuntimeMonitorSpec.scala

```diff
@@ -303,6 +311,47 @@ class BaseCloudServiceRuntimeMonitorSpec extends AnyFlatSpec with Matchers with
     res.unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
   }
 
+  it should "detach Ready disk on failed runtime create" in isolatedDbTest {
```

Comment:
I am thinking, should we also detach the PD when the runtime is in deleting status? not just deleted

Reply:
that's done as a part of the deletedRuntime func in the [GceRuntimeMonitor](https://github.com/DataBiosphere/leonardo/blob/4ad7058eff65dbe4500185b9979b9c1c2d06fd6b/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/GceRuntimeMonitor.scala#L401) (complete deletion detaches the disk)